apps/demos/src/app/home/home.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'rxa-home',
+  template: `
+    <h1>Welcome to RxAngular Demos!</h1>
+    <h3>Explore various sections that will guide you through features of RxAngular.</h3>
+
+    <ul>
+      <li><h4>🏁 Fundamentals</h4></li>
+      Demos presenting fundamental rules and functionalities existing in Angular that are good to know before you start your journey with RxAngular.
+
+      <li><h4>🧰 Template</h4></li>
+      Playground for different functionalities of the <code>@rx-angular/template</code> package.
+
+      <li><h4>📋 Tutorials</h4></li>
+      Tutorials for features existing in the <code>@rx-angular</code> packages.
+
+      <li><h4>🧮 Integrations</h4></li>
+      ???
```

Comment:
Rx-Angular/state integrated with different libs and patterns? We will have the heroes demo, the example with global state, and  others there